src/index.js

```diff
@@ -1,6 +1,6 @@
 import fs from 'fs'
 
-export default function (pluginOptions) {
+export default function (path, pluginOptions) {
```

Comment:
The variable "path" not used in next code. I think, you add it by mistake

Reply:
I had to make `pluginOptions` the second argument, otherwise it didn't work. I'm using babel `7.0.0-beta.53 (@babel/core 7.0.0-beta.53)`.

Reply:
```suggestion
export default function (path, pluginOptions) {
export default function (...args) {
```
Maybe do this?